assets/src/domain/eventEditor/ui/datetimes/datesList/actionsMenu/dateEntityMainMenuItem/CopyDate.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { omit } from 'ramda';
+import { __ } from '@wordpress/i18n';
+
+import { Copy } from '@application/ui/layout/entityActionsMenu/entityMenuItems';
+import { useDatetimeContext } from '@edtrHooks/index';
+import { useDatetimeItem } from '@edtrServices/apollo/queries';
+import { useDatetimeMutator } from '@edtrServices/apollo/mutations';
+
+const CopyDate = (props) => {
+	const { id } = useDatetimeContext();
+	const datetime = useDatetimeItem({ id });
+	const { createEntity } = useDatetimeMutator();
+	const newDatetime = omit(
+		[
+			'dbId',
+			'id',
+			'isActive',
+			'isExpired',
+			'isSoldOut',
+			'isTrashed',
+			'isUpcoming',
+			'length',
+			'status',
+			'__typename',
+		],
+		datetime
+	);
```

Comment:
IMHO, it's better and safe o use `pick` instead of `omit`, just in case any other field is added in future.

Reply:
that's a double edged sword because you might want to also copy those properties as well...
impossible to know, but yours is likely the safer option because you have to explicitly add fields you want